src/plot.cpp

```diff
@@ -19,6 +19,61 @@
 
 using namespace std;
 
+const char *gMarkerVertexShaderSrc =
+"#version 330\n"
+"in vec2 point;\n"
+"uniform mat4 transform;\n"
+"void main(void) {\n"
+"   gl_Position = transform * vec4(point.xy, 0, 1);\n"
+"   gl_PointSize = 10;\n"
+"}";
+
+
+const char *gMarkerSpriteFragmentShaderSrc =
```

Comment:
We can improve this with GLSL subroutines that can be selected by the program instead of the switch statement inside shader.